miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -964,7 +879,7 @@ def set_mop_mode(self, mop_mode: MopMode):
     @command()
     def mop_intensity(self) -> MopIntensity:
         """Get mop scrub intensity setting."""
-        if self.model != ROCKROBO_S7:
+        if self.model not in [ROCKROBO_S7, ROCKROBO_S7_MAXV]:
             raise UnsupportedFeatureException(
                 "Mop scrub intensity not supported by %s", self.model
```

Comment:
An idea: we could remove the model check and embed this also to the returned status. This can be done later in a separate PR though, that should also convert the `embed()` to take a partial to execute instead of a prepared status container. The reasoning is that we can then make error handling inside the embed (and potentially skip unsupported methods from future requests).

Reply:
It is already in the returned status (water_box_mode is included in the get_status command).
(So this seperate method is a bit redundant).

But you are totally right that there schould be a general way to handle commands that are only supported by a subset of the models.
But lets do that in a future PR.

Reply:
Ah, sorry, I didn't notice that. Sounds good to me! :+1: